spinta/exceptions.py

```diff
@@ -1001,4 +1001,4 @@ class InvalidClientFileFormat(UserError):
 
 
 class MissingRefModel(UserError):
-    template = "Property of type `ref` or `backref` should have a model in the `ref` column"
+    template = 'Property "{property_name}" of type "{property_type}" in the model "{model_name}" should have a model name in the `ref` column, to which it refers.'
```

Comment:
```suggestion
    template = 'Property "{dtype.prop.place}" of type "{dtype.name}" in the model "{dtype.prop.model.name}" should have a model name in the `ref` column, to which it refers.'
```

Reply:
It actually was a bit different, through the use of `context`, but I changed it.